python/pyarrow/parquet.py

```diff
@@ -560,6 +560,15 @@ def _sanitize_table(table, new_schema, flavor):
     enabled, then dictionary is preferred.
     The byte_stream_split encoding is valid only for floating-point data types
     and should be combined with a compression codec.
+col_encoding : dict, default None
+    Specify the encoding scheme on a per column basis.
+    Valid values: {'PLAIN', 'BIT_PACKED', 'RLE', 'BYTE_STREAM_SPLIT',
+    'DELTA_BINARY_PACKED', 'DELTA_BYTE_ARRAY'}
+    Unsupported encodings: DELTA_LENGTH_BYTE_ARRAY, PLAIN_DICTIONARY and
+    RLE_DICTIONARY. Last two options are already used by default.
+    Certain encodings are only compatible with certain data types.
+    Please refer to the encodings section of Reading and writing Parquet
+    files <https://arrow.apache.org/docs/cpp/parquet.html#encodings>_.
```

Comment:
```suggestion
    files `<https://arrow.apache.org/docs/cpp/parquet.html#encodings>`__.
```

(Restructuredtext needs the backticks around it ..)